tests/test_completion.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import shlex
+
+commands_without_input = shlex.split(
+    """
+        quit true false reload version
+        list_commands list_monitors
+        list_rules list_keybinds
+        lock unlock
+        close_or_remove close_and_remove
+        remove rotate
+        mouseunbind
+        use_previous
+        lock unlock
+    """)
+
+
+@pytest.mark.parametrize('name', commands_without_input)
+def test_inputless_commands(hlwm, name):
+    assert hlwm.call_xfail_no_output('complete 1 ' + name) \
+        .returncode == 7
```

Comment:
I'd advise against comparing it to a numeral that is not really defined/hardcoded/documented. That's going to be a pain point if we ever touch that exit code enum.

We should to at least one of these:
 * add a FIXME here explaining where the `7` comes from and that it's somewhat fragile
 * explicitly assign a numeral to each `HERBST_` exit code to clarify that they are assigned for all eternity (if that's our intention)
 * make the enum values available in the tests, so that we can use them right here instead of numerals

Reply:
(Btw: The man page for herbstclient refers to that of herbstluftwm for a list of exit codes, but I couldn't find it there. In case I just missed it, strike my mention of "undocumented".)

Reply:
I've opened #253 for the lack of documentation and added a FIXME here.

Reply:
Oh and: If we're going with the eternal assignments (which is probably the consequence of documenting them), I'd say we could simply duplicate them in conftest.py. I've mentioned technical solutions for this before (you could parse ipc-protocol.h using ctypeslib's clang2py), but I don't think they're worth the effort in that case.